homeassistant/components/openhome/__init__.py

```diff
@@ -1 +1,51 @@
 """The openhome component."""
+
+import asyncio
+import logging
+
+import aiohttp
+from async_upnp_client.client import UpnpError
+from openhomedevice.device import Device
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .media_player import OpenhomeDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
+
+
+async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Unload a config entry."""
+    # Forward to the same platform as async_setup_entry did
+    return await hass.config_entries.async_unload_platforms(config_entry, PLATFORMS)
```

Comment:
You need to also remove the device from `hass.data[DOMAIN]`

Reply:
Done